spec/person_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../person'
+
+describe Person do
+  before :each do
+    @person = Person.new('Alice', 15)
+  end
+
+  it 'should have a name' do
+    expect(@person.name).to eq('Alice')
+  end
+
+  it 'should have an age' do
+    expect(@person.age).to eq(15)
+  end
+end
```

Comment:
As mentioned in the requirements, you should add unit tests for methods that exist in the Person class: 
- generate_new_id
- add_rental
- correct_name
- can_use_services?